backend/onyx/db/users.py

```diff
@@ -185,3 +192,43 @@ def batch_add_ext_perm_user_if_not_exists(
     db_session.commit()
 
     return found_users + new_users
+
+
+def delete_user_from_db(
+    user_to_delete: User,
+    db_session: Session,
+) -> None:
+    for oauth_account in user_to_delete.oauth_accounts:
```

Comment:
deleting users is often kind of tricky, we need to delete everything related to that user no? Like chat sessions, files in the blob store, etc?